common/djangoapps/philu_commands/management/commands/send_referral_follow_up_email.py

```diff
@@ -5,35 +5,38 @@
 from django.utils import timezone
 
 from openedx.features.smart_referral.models import SmartReferral
-from openedx.features.smart_referral.tasks import task_send_referral_follow_up_email
+from openedx.features.smart_referral.tasks import task_send_referral_follow_up_emails
 
 CONTACT_EMAIL = 'contact_email'
 DAYS_TO_SEND_FOLLOW_UP_EMAIL = 3
 
 
 class Command(BaseCommand):
     help = """
-    Send follow up referral email to all those contacts
+    Send follow-up referral email to all those contacts who are at-least 3 days old, contacts which aren't registered
+    on our platform and we hadn't send them follow-up email before.
```

Comment:
The sentence has some errors. It can be rephrased to:

"Send follow-up referral email to all those contacts which are at least 3 days old, are not registered on our platform, and have not received follow-up email."